navigators/AuthNavigator.tsx

```diff
@@ -0,0 +1,48 @@
+import {useColorScheme} from 'react-native';
+import type {NavigationProp} from '@react-navigation/native';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+
+import {getAppColorStyles} from '../utils/styles/colors';
+import {getAppColors} from '../utils/colors';
+import HomeScreen from '../screens/auth/home/HomeScreen';
+import DetailedMuseumScreen from '../screens/auth/DetailedMuseumScreen';
+import type Museum from '../model/Museum';
+import {FavouritesProvider} from '../providers/FavouritesProvider';
+
+export type AuthRootStackParamList = {
+  Home: undefined;
+  DetailedMuseum: {museum: Museum};
+};
+
+export type AuthStackNavigation = NavigationProp<AuthRootStackParamList>;
+
+const Stack = createNativeStackNavigator<AuthRootStackParamList>();
+export default function AuthNavigator() {
+  const colorScheme = useColorScheme();
+  const {backgroundStyle} = getAppColorStyles(colorScheme);
+  const {backgroundColor, headerColor} = getAppColors(colorScheme);
+
+  return (
+    <FavouritesProvider>
+      <Stack.Navigator
+        initialRouteName="Home"
+        screenOptions={() => ({
+          tabBarStyle: backgroundStyle,
+          headerStyle: {backgroundColor: backgroundColor},
```

Comment:
Pro tip for JS objects 😄 
```suggestion
          headerStyle: {backgroundColor},
```